cluster/images/etcd/rollback/rollback.go

```diff
@@ -42,7 +44,7 @@ import (
 	"github.com/golang/glog"
 )
 
-const rollbackVersion = "2.3.7"
+const rollbackVersion = "2.2.0"
```

Comment:
I don't understand. This too is supposed to rollback from 3.0.* format to 2.3.7. Why do we need any information about 2.2.0?